<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QDialog" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>60</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #696969;</string>
   </property>
   <property name="text">
    <string>Реєстрація</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_To_Authorization">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: transparent;
	background-image: url(:/resources/png/Icon60x60.png);
}

QPushButton:hover {
	background: transparent;
	background-image: url(:/resources/png/IconGr60x60.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="FinishRagistration">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>380</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 10px;
  text-decoration: none;
  width: 140px;
  height: 45px;
  line-height: 45px;
  border-radius: 45px;
  margin: 10px 20px;
  text-transform: uppercase;
  text-align: center;
  letter-spacing: 3px;
  font-weight: 600;
  color: #524f4e;
  background: white;
border-radius: 10px;
}
QPushButton:hover {
border-radius: 10px;
  background: #2EE59D;
  color: white;
}</string>
   </property>
   <property name="text">
    <string>Зареєструватись</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Email">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>200</y>
     <width>211</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px sold #2EE59D;
	border-radius: 20px;
	color: #524f4e;
	padding-left: 20px;
	padding-right: 20px;
	background-color: white;
}
QLineEdit:hover {
	border: 2px solid #2EE59D
}
QLinneEdit:focus {
	border: 2px solid rgb(42, 44, 54)
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RegPass">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>260</y>
     <width>211</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px sold #2EE59D;
	border-radius: 20px;
	color: #524f4e;
	padding-left: 20px;
	padding-right: 20px;
	background-color: white;
}
QLineEdit:hover {
	border: 2px solid #2EE59D;
}
QLinneEdit:focus {
	border: 2px solid #2EE59D
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RegPass1">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>320</y>
     <width>211</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px sold #2EE59D;
	border-radius: 20px;
	color: #524f4e;
	padding-left: 20px;
	padding-right: 20px;
	background-color: white;
}
QLineEdit:hover {
	border: 2px solid #2EE59D
}
QLinneEdit:focus {
	border: 2px solid rgb(42, 44, 54)
} </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RegLogin">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>140</y>
     <width>211</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px sold #2EE59D;
	border-radius: 20px;
	color: #524f4e;
	padding-left: 20px;
	padding-right: 20px;
	background-color: white;
}
QLineEdit:hover {
	border: 2px solid #2EE59D
}
QLinneEdit:focus {
	border: 2px solid rgb(42, 44, 54)
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>470</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #696969;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="RegPassCheck">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Показати пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_To_Authorization_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: transparent;
	background-image: url(:/resources/png/Icon60x60.png);
}

QPushButton:hover {
	background: transparent;
	background-image: url(:/resources/png/IconGr60x60.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
